Servers/driver/autoDriver.driver.ts

```
@@ -485,39 +461,268 @@ const insertQuery: TableList = [
+    generateValuesString: roleGenerateValuesString,
+  } = insertQuery["roles"];
+  let roles;
+  if (roleMockData.length !== 0) {
+    const values = roleMockData.map((d) => roleGenerateValuesString(d as any));
+    insertString += values.join(",") + "RETURNING id;";
+    roles = await insertData(insertString as string);
+  }
+
+  var {
+    mockData: userMockData,
+    tableName,
```

Biome (1.9.4)

[error] Shouldn't redeclare 'tableName'. Consider to delete it or rename it.

'tableName' is defined here:


(lint/suspicious/noRedeclare)